diff --git a/node_modules/expo-modules-core/android/src/main/cpp/worklets/WorkletJSCallInvoker.cpp b/node_modules/expo-modules-core/android/src/main/cpp/worklets/WorkletJSCallInvoker.cpp
index a2d707c..0a69a24 100644
--- a/node_modules/expo-modules-core/android/src/main/cpp/worklets/WorkletJSCallInvoker.cpp
+++ b/node_modules/expo-modules-core/android/src/main/cpp/worklets/WorkletJSCallInvoker.cpp
@@ -24,7 +24,7 @@ namespace expo {
       return;
     }
 
-    workletRuntime->executeSync([func = std::move(func)](jsi::Runtime &rt) -> jsi::Value {
+    workletRuntime->runSync([func = std::move(func)](jsi::Runtime &rt) -> jsi::Value {
       func(rt);
       return jsi::Value::undefined();
     });
